### PR TITLE
Project Specification

### PR DESCRIPTION
**Pull request by LeandroLM**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
```
Comment (jeanpsv) [inline .travis.yml]: I've removed this file from other projects in this block, because it do nothing

File: README.md [hunk 2]
```diff
@@ -1 +1,212 @@
...
+
+### 1 - A página deve possuir um título centralizado com o nome do seu jogo
+
+O título deverá conter o **id** `title`.
+
+### 2 - A página deve conter um texto centralizado com o ***RGB*** a ser adivinhado
+
+O texto deverá conter os três números das cores RGB a ser adivinhada, separados por vírgula e entre parênteses, conforme ilustrado na imagem acima. Exemplo: `(168, 34, 248)`.
+
+O elemento que contém o texto deverá deverá conter o **id** `rgb-color`.
```
Comment (jeanpsv) [inline README.md]:
```suggestion
O elemento que contém o texto deverá conter o **id** `rgb-color`.
```

File: README.md [hunk 3]
```diff
@@ -1 +1,212 @@
...
+
+## BÔNUS
+
+### 7 - Crie um placar que incremente 3 pontos para cada acerto no jogo
+
+No início do jogo, o placar deve ser zero. A cada acerto, o placar deve ser incrementado em 3 pontos.
+
+Ao clicar no botão reiniciar, o placar **NÃO** deve ser resetado para zero.
+
+O elemento que conterár o placar deve ter o **id** `score`.
```
Comment (jeanpsv) [inline README.md]:
```suggestion
O elemento que conterá o placar deve ter o **id** `score`.
```

File: README.md [hunk 4]
```diff
@@ -1 +1,212 @@
...
+
+* `Erro 3:` -2 pontos.
+
+* `Erro 4:` -3 pontos.
+
+* `Acerto 1:` +3 pontos.
+
+* `Erro 5:` -1 pontos.
+
+Exemplo: Joana tem 3 acertos consecutivos seguidos de 4 erros e por fim mais 2 acertos. A pontuação de Jonas é de 9 pontos (3 + 3 + 4 - 1 - 1 - 2 - 3 + 3 + 3).
```
Comment (jeanpsv) [inline README.md]:
```suggestion
Exemplo: Joana tem 3 acertos consecutivos seguidos de 4 erros e por fim mais 2 acertos. A pontuação de Joana é de 9 pontos (3 + 3 + 4 - 1 - 1 - 2 - 3 + 3 + 3).
```

File: .travis.yml (hunk 1 above)
Comment (LeandroLM) [reply .travis.yml]: Oh yeah, I removed it too but forgot to do it here 😬

**Pull request merged**